/**:
  ros__parameters:
    camera_name: "camera_2"
    frame_id: "camera_left/camera_link"
    pixel_format: bgr8
    serial_no: 210900909
    fps: 15
    horizontal_binning: 4
    vertical_binning: 4
    resize_image: true
    camera_info_url: "file:///home/volt/projects/volt_drivers_ws/src/arena_camera/config/camera_left_quarter.yaml"
